packages/calypso-e2e/src/lib/components/editor-popover-menu-component.ts

```diff
@@ -0,0 +1,34 @@
+import { Page, Locator } from 'playwright';
```

Comment:
Same with this file -- not actually used, but I figure future tests might, so we can leave in?

Again, if you want, I can gut as needed!

Reply:
Is this for popover like this?
<img width="351" alt="image" src="https://user-images.githubusercontent.com/6549265/162446523-b105421c-a3f6-4815-9fbd-04657b4d3038.png">

Reply:
Yes, correct!